### PR TITLE
CLOUDP-196436 Check podman installation and machine init/start

### PR DESCRIPTION
**Pull request by fmenezes**
## Proposed changes

Check podman installation and run `podman machine init` and `podman machine start`

_Jira ticket:_ CLOUDP-196436

## Checklist

- [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/CONTRIBUTING.md) (if appropriate)
- [ ] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [ ] I have updated [test/README.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/README.md) (if an e2e test has been added)
- [ ] I have run `make fmt` and formatted my code

### CODE REVIEW

File: internal/cli/atlas/deployments/setup.go [hunk 1]
```diff
@@ -46,9 +46,12 @@ const (
 	clusterNamePattern = "^[a-zA-Z0-9][a-zA-Z0-9-]*$"
 )
 
+var ErrPodmanNotReady = errors.New("podman not found in your system, check requirements at http://docpage")
```
Comment (blva) [inline internal/cli/atlas/deployments/setup.go:49]: will this be podman link itself of a DOP page if so maybe we can create the JIRA?

**Review by blva — APPROVED** [review]
LGTM

File: internal/cli/atlas/deployments/setup.go (hunk 1 above)
Comment (fmenezes) [reply internal/cli/atlas/deployments/setup.go:49]: I think this has to be a docsp page (ours)

**Comment by fmenezes** [comment]
evergreen status

**Pull request merged**